apps/staking/src/hooks/use-api.tsx

```diff
@@ -141,6 +144,10 @@ const useApiContext = (hermesUrl: string) => {
   const { isMainnet } = useNetwork();
   const { mutate } = useSWRConfig();
   const hermesClient = useMemo(() => new HermesClient(hermesUrl), [hermesUrl]);
+  const pythnetClient = useMemo(
+    () => new PythnetClient(new Connection("https://pythnet.rpcpool.com")),
```

Comment:
make this env variable?

Reply:
Shouldn't we use the same as `MAINNET_RCP`?  Why use a different rpcpool subdomain here?

If we do need a different subdomain, my suggestion is to put this in an env var and export it in `config/server.ts` and use the utilities (`getOr`) in there.  Then, pass it down via `ApiProvider` in `Root/index.tsx` just like `hermesUrl`.

Reply:
It's not the same network. This is Pythnet

Reply:
oh d'oh of course